modules/core/arrow-core-data/src/test/kotlin/arrow/core/ListKTest.kt

```diff
@@ -33,6 +39,8 @@ class ListKTest : UnitSpec() {
 
     testLaws(
       ShowLaws.laws(ListK.show(), eq) { listOf(it).k() },
+      SemigroupLaws.laws(ListK.semigroup(), listOf(1, 2, 3).k(), listOf(4, 5, 6).k(), listOf(7, 8, 9).k(), eq),
+      MonoidLaws.laws(ListK.monoid(), Gen.listK(Gen.int()), ListK.eq(Int.eq())),
```

Comment:
MonoidLaws is implemented to also include SemigroupLaws, so this shouldn't be necessary :D

Reply:
I knew that but I saw existing tests using both. I will update them as well